runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -51,6 +48,11 @@
 import net.runelite.client.plugins.xptracker.XpTrackerPlugin;
 import net.runelite.client.ui.overlay.OverlayManager;
 import net.runelite.client.util.QueryRunner;
+import java.util.ArrayList;
```

Comment:
Do not move imports to bottom (see https://github.com/runelite/runelite/wiki/Code-Conventions)